src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -375,6 +506,19 @@ public static TextBuilder builder() {
     }
 
     /**
+     * Creates a new unformatted {@link TextBuilder.Placeholder} with the
+     * specified textArgument.
+     *
+     * @param textArgument The textArgument of the placeholder
+     * @return The created placeholder builder
+     * @see Text.Placeholder
+     * @see TextBuilder.Placeholder
+     */
+    public static TextBuilder.Placeholder placeholderBuilder(TextArgument textArgument) {
```

Comment:
Yes, that true.

But:
`Texts.of(String text)`, 
thats why that method is called `Texts.placeholder(String key)` 
thats why I named `Texts.placeholder(TextArgument argument)` this way.